core/src/test/scala/org/apache/spark/sql/delta/DeltaLogSuite.scala

```diff
@@ -505,4 +508,54 @@ class DeltaLogSuite extends QueryTest
       )
     }
   }
+
+  test("DeltaLog should throw exception if spark.sql.extensions and " +
```

Comment:
Could you move these tests to `DeltaExtensionAndCatalogSuite.scala`? This would make new tests added after these tests in future work.